### ISSUE TITLE
Xcode 11 platformFilter isn't understood for PBXTargetDependency

### ISSUE THREAD

**Issue by lickel** [CocoaPods/Xcodeproj#693]
When Catalyst is enabled for an iOS app which has targets that are invalid on iOS, the target automatically gets a `platformFilter`. 

This is _not_ fixed by https://github.com/CocoaPods/Xcodeproj/pull/687

```
[!] [!] Xcodeproj doesn't know about the following attributes {"platformFilter"=>"ios"} for the 'PBXTargetDependency' isa.
```

```
		329B71611D82926D00336078 /* PBXTargetDependency */ = {
			isa = PBXTargetDependency;
			platformFilter = ios;
			target = 329B71551D82926C00336078 /* NotificationContentReaction */;
			targetProxy = 329B71601D82926D00336078 /* PBXContainerItemProxy */;
		};
```

**Comment by dnkoutso** [comment]
@lickel thanks! We can open up a PR against https://github.com/CocoaPods/Xcodeproj.

**Comment by dnkoutso** [comment]
PR was merged. Will ship with 1.11.0 of Xcodeproj. Use https://bundler.io/ to point to a specific SHA or branch that includes the fix.

**Comment by dnkoutso** [comment]
Released with 1.11.0.
